Makefile.ci

```diff
@@ -5,7 +5,11 @@ CI_CONTAINER_IIDFILE= ./build/ci-output/container.txt
 # Note that for check-generate to work, the directory must be named
```

Comment:
Do we even need the buildkite CI anymore? I thought we were building in GitHub now - but I haven't been paying the closest of attention?

Reply:
I'm using Buildkite to run additional tests contained in a separate repo. In order for that to work, I need to build the Provider and make it available. It seems simplest to me to have a small build job that creates the Provider and saves it as an Artifact that is then readily accessible to download and test. Maybe you can let me know if this will cause problems in some way.